packages/lesmis-server/src/connections/EntityConnection.ts

```diff
@@ -4,12 +4,15 @@
  *
  */
 import camelcaseKeys from 'camelcase-keys';
-import { QueryParameters } from '@tupaia/server-boilerplate';
+import { QueryParameters } from '@beyondessential/tupaia-api-client';
```

Comment:
I decided to move all of the api-client logic out of server-boilerplate so that devs wouldn't add more code there. So we have some quite unsightly imports used in deprecated classes.